### PR TITLE
Fixed wrong succeed on job retries

### PR DESCRIPTION
**Pull request by RodriFS**

### CODE REVIEW

File: src/Helpers/JobTypes.cs [hunk 1]
```diff
@@ -89,21 +89,19 @@ public static async Task Execute(IJobExecutionContext context, Func<Task> callba
 
         var trigger = context.Trigger as SimpleTriggerImpl;
 
-        if (trigger!.TimesTriggered >= intervals.Length)
+        if (trigger!.TimesTriggered <= intervals.Length)
         {
-            return;
+            var repeatInterval = intervals[trigger!.TimesTriggered - 1];
```
Comment (Jossec101) [inline src/Helpers/JobTypes.cs:94]: If the interval is a collection you can use collection.Last() or even Collection.LastOrDefault() instead of indexes

Comment (RodriFS) [reply src/Helpers/JobTypes.cs:94]: unfortunatelly, I'm not consuming the array, I'm just traversing it. Each time that it runs, it gets the trigger count and so it gets the next of the array.
Another way would be to consume the first of the array, then save the array in the job data map again. But last time I tried to update a value inside a job, it didn't work

Comment (Jossec101) [reply src/Helpers/JobTypes.cs:94]: LGTM

**Review by Jossec101 — APPROVED** [review]
LGTM

**Pull request merged**